README.md

```diff
@@ -12,13 +12,21 @@ Program queries logger modbus port in infinite loop and sends data into MQTT top
 7. Run `./sofar` or `sofar-arm`
 
 ## Output data format
+### MQTT
 Data will be sent into MQTT topic with name `{mqttPrefix}/{fieldName}` where:
 * mqttPrefix is value defined in `config.yaml` e.g. `/sensors/energy/inverter`
 * fieldName is measurement name, all available measurements are described in `adapters/devices/sofar/sofar_protocol.go`, e.g. `PV_Generation_Today`
 
 Full topic name for given example values is `/sensors/energy/inverter/PV_Generation_Today`.
 Additional field is `All` which contains all measurements and their values marshalled into one json.
 
+### OTLP
+Data can also be sent over OTLP protocol to a gRPC or http server. Typically, this would be received by the 
+[OTel-Collector](https://opentelemetry.io/docs/collector/) for further export to any required platform. 
+
+Metrics are all captured as gauges and recorded and exported at the same frequency that measurements are taken. 
+Metric names follow the convention `solar.logging.<fieldName>`.
```

Comment:
This is still wrong. Default value is 'sofar.logger.'

Reply:
Can you double check this again? I think you might be looking at an outdated version of the file. This is what I can see: https://github.com/kubaceg/sofar_g3_lsw3_logger_reader/pull/7/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R28

Reply:
Well, I see still solar.logging instead of sofar.loggar
<img width="757" alt="image" src="https://user-images.githubusercontent.com/696884/226344935-1d569c8e-358a-4d9e-8297-5f2ba1baad5b.png">

Reply:
I'm confused by your comment and screenshot. To me, everything says sofar. There is no mention of solar.